charts/rapid-response/values.yaml

```diff
@@ -164,3 +164,19 @@ rapidResponse:
     key: node-role.kubernetes.io/master
   - effect: NoSchedule
     key: node-role.kubernetes.io/control-plane
+
+rbac:
+  # true here enables creation of rbac resources
+  create: true
+
+serviceAccount:
+  # true here enables creation of service account
+  create: true
+  # Use this value as kspmCollectorServiceAccountName
+  name: "rapid-response"
```

Comment:
@yashgiri since these are test values, I think would be worth having them inside this file:
https://github.com/sysdiglabs/charts/blob/master/charts/rapid-response/ci/test-values.yaml.template